### PR TITLE
Reformat elementwise expm1

### LINKED ISSUE #1874: expm1

### PR DESCRIPTION
**Pull request by hattiq**
Closes #1874

### CODE REVIEW

**Review by chie2727 — CHANGES_REQUESTED** [review]
Hi @hattiq , thanks for starting the task!
Please follow the guidelines in the Docstrings Examples guide and add in all the required examples:

https://github.com/unifyai/ivy/blob/master/docs/partial_source/deep_dive/12_docstring_examples.rst

**Comment by hattiq** [comment]
Thanks, @chie2727 , I tried to add all relevant examples (addressed in points 1 through 11 in docs) x. Can you please point out?

**Review by chie2727 — CHANGES_REQUESTED** [review]
Hi @hattiq , thanks for making all the changes!

**Comment by hattiq** [comment]
Hi @chie2727, I made the changes and also added special case examples in docstrings. Please review!

**Comment by hattiq** [comment]
reason and fix for failing docstring test #1944

File: ivy/functional/ivy/elementwise.py [hunk 1]
```diff
@@ -1530,6 +1530,68 @@ def expm1(
         an array containing the evaluated result for each element in ``x``. The returned
         array must have a floating-point data type determined by :ref:`type-promotion`.
 
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.elementwise_functions.add.html>`_ # noqa
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:1535]: Please update the link on line 1535 to the correct one for `expm1`

**Review by chie2727 — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/elementwise.py (hunk 1 above)
Comment (hattiq) [reply ivy/functional/ivy/elementwise.py]: Done.

File: ivy/functional/ivy/elementwise.py [hunk 2]
```diff
@@ -1530,6 +1530,68 @@ def expm1(
...
+    
+    With :code:`ivy.NativeArray` inputs:
+    
+    >>> x = ivy.native_array([[1], [5], [-ivy.inf]])
+    >>> ivy.expm1(x)
+    ivy.array([[  1.72],
+       [147.  ],
+       [ -1.  ]])
+    
+    With :code:`ivy.Array` Method:
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:1563]: Last change to make: on line 1563 please rename `method` to `instance method`

**Review by chie2727 — CHANGES_REQUESTED** [review]

**Review by chie2727 — APPROVED** [review]
Thanks for making the changes. I'll merge your changes in now!

**Pull request merged**